src/Interfaces/LazySet.jl

```diff
@@ -248,6 +249,30 @@ Alias for the support vector σ.
 """
 const support_vector = σ
 
+"""
+    isboundedtype(::Type{<:LazySet})
+
+Determine whether a set type only represents bounded sets.
+
+### Input
+
+- `LazySet` -- set type for dispatch
+
+### Output
+
+`true` if the set type only represents bounded sets.
+Note that some sets may still represent an unbounded set even though their type
+actually does not (example: [`HPolytope`](@ref)).
```

Comment:
```suggestion
actually does not (example: [`HPolytope`](@ref), because the constructor doesn't assert boundedness by default).
```

Reply:
I will write `, because the construction with non-bounding linear constraints is allowed`.

Reply:
:+1:  thanks